sequencer/api/state_relay_server.toml

```diff
@@ -0,0 +1,19 @@
+[meta]
+NAME = "hotshot_state_relay_server"
+DESCRIPTION = "A relay server for storing and serving the light client states and their signatures"
+FORMAT_VERSION = "0.1.0"
+
+[route.poststatesignature]
+PATH = ["state/post"]
+METHOD = "POST"
+DOC = """
+Post a light client state and its signature for a given block height.
+"""
+
+[route.getlateststate]
+PATH = ["state/getlatest"]
```

Comment:
Same nit: could just be GET `state`

Reply:
changed in [1dde57c](https://github.com/EspressoSystems/espresso-sequencer/pull/1016/commits/1dde57c61e4683b1fde67bb5bde9e666a563311b)